server/database/queries/getExercise.js

```diff
@@ -0,0 +1,5 @@
+const connection = require('../config/connection');
+
+const getExercises = () => connection.query('SELECT * FROM exercises');
```

Comment:
Zein, This query will not fit our front , you will not show the met value to the user, is important to show calories/hr for each exercise so you need here to get the user weight and multiply be the met value.
Also, we need to make a check on the user exercise, so also you need the userExersices Table also.
I think we need to make the following three queries and then handle it's in the controller and send them back to the front.

```sql
const userWeight = connection.query({
    text: 'SELECT weight from users where id=$1',
    values: [userId],
  });

  const exercises = connection.query('SELECT * FROM exercises');

  const userExercises = connection.query(
    'SELECT id,exercises_id ,exercise_duration_in_minutes as duration from UserExerciseRelation where users_id=$1 and created_at=current_date',
    [userId],
  );
```